packages/cli/src/util/telemetry/commands/teams/invite.ts

```diff
@@ -1,11 +1,16 @@
 import { TelemetryClient } from '../..';
+import type { TelemetryMethods } from '../../types';
+import type { inviteSubcommand } from '../../../../commands/teams/command';
 
-export class TeamsInviteTelemetryClient extends TelemetryClient {
-  trackCliArgumentEmail(count: number) {
-    if (count > 0) {
+export class TeamsInviteTelemetryClient
+  extends TelemetryClient
+  implements TelemetryMethods<typeof inviteSubcommand>
+{
+  trackCliArgumentEmails(value: string | undefined) {
+    if (value) {
       this.trackCliArgument({
         arg: 'email',
-        value: count === 1 ? 'ONE' : 'MANY',
+        value,
```

Comment:
I understand that you changed this to conform to the method signature generated by `implements TelemetryMethods<typeof inviteSubcommand>`, but I don't like the redaction isn't inside the track method anymore, and therefore can be missed during a future code change.

Reply:
0de3c4dec52328e35349b86815f59d1f4f4ae25b